src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +19,55 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const findAllCustomerSerializer: CustomerFindAllRequestHandler = (req, res, next) => {
+    const { customersFound, limit } = res.locals;
+    
+    customersFound.forEach(customer => {
+        res.locals.customerToRespond.push({
```

Comment:
Esse código dará erro, pois `res.locals.customerToResponse` tem por padrão o valor `undefined`.

Não tem como encadear o método `push`, pois ele não existe.

Caso quisesse seguir essa estratégia, teria que atribuir um array vazio ao mesmo.
```suggestion
    res.locals.customerToRespond = [];
    
    customersFound.forEach(customer => {
        res.locals.customerToRespond.push({
```